### ISSUE TITLE
Update links in submission emails 

### ISSUE THREAD

**Issue by caitmuenster** [mozilla/addons-server#16650]
When developers submit an add-on to be listed on AMO, they receive [this email](https://github.com/mozilla/addons-server/blob/master/src/olympia/devhub/templates/devhub/emails/submission.html). 

The following links need to be updated: 

* `https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews` should be changed to `https://extensionworkshop.com/documentation/publish/add-on-policies/`

* `https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Resources_for_publishers` should be changed to `https://extensionworkshop.com/documentation/manage/resources-for-publishers/`

* Line 15 of the email should be deleted. 

If this is your first contribution, please refer to https://wiki.mozilla.org/Add-ons/Contribute/Code on how to get started. 

Mentor: @eviljeff

**Comment by Mitrajit** [comment]
I want to take up this issue. Assign it to me.

**Comment by eviljeff** [comment]
> I want to take up this issue. Assign it to me.

You can go ahead and start work on it.  We will assign the issue once the PR is ready.

**Comment by Mitrajit** [comment]
PR is ready.

**Comment by AlexandraMoga** [comment]
The new email copy sent for first time add-on submissions has been updated:

![image](https://user-images.githubusercontent.com/31961530/110139669-f6642480-7ddb-11eb-948a-0b9e45de9b71.png)

@eviljeff I've noticed that the changes are already on stage, although the issue was closed for this milestone.

**Comment by eviljeff** [comment]
@AlexandraMoga that's super weird!  I double-checked and stage amo still has the 2021-03-04 tag, which doesn't include this patch.  (I'll investigate further on Monday)

**Comment by AlexandraMoga** [comment]
Also, I've noticed that the email sent still includes this line (both -dev and stage) :
`If you are interested in getting more involved with Mozilla and want to help keep the add-ons ecosystem safe and healthy, please consider becoming a volunteer add-on reviewer. You can learn more at https://wiki.mozilla.org/Add-ons/Reviewers`

If I'm looking at the [template](https://github.com/mozilla/addons-server/blob/master/src/olympia/devhub/templates/devhub/emails/submission.html), that line is no longer present.

Examples: [dev email](https://addons-internal.dev.mozaws.net/en-US/admin/models/amo/fakeemail/96104/change/), [stage email](https://addons-internal.stage.mozaws.net/en-US/admin/models/amo/fakeemail/64001/change/)

**Comment by eviljeff** [comment]
Ah.  So, there are two versions of the email template - one for plaint text, one for html.  If you look further down the stage email you'll see the html has the old links.  (#13561 last year overlooked changing both the txt and html templates)

**Comment by AlexandraMoga** [comment]
> Ah. So, there are two versions of the email template - one for plaint text, one for html. If you look further down the stage email you'll see the html has the old links. (#13561 last year overlooked changing both the txt and html templates)

Ah, yes. That was confusing. I'll file a low priority issue to make the submission `.html` and `.txt` templates even, to avoid such confusions in the future.
